packages/cashier/src/pages/payment-agent/payment-agent-deposit-details/payment-agent-deposit-details.tsx

```diff
@@ -21,33 +21,22 @@ const PaymentAgentDepositDetails = ({ payment_agent }: TPaymentAgentDepositDetai
         );
     };
 
-    const PaymentAgentEmailDetails = () => {
+    const PaymentAgentTransferLimitDetails = () => {
         return (
-            <PaymentAgentDetail
-                action='mailto'
-                icon='IcEmailOutlineNew'
-                rel='noopener noreferrer'
-                target='_blank'
-                has_red_color
-                title={localize('Email')}
-            >
-                {payment_agent.email}
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMinimumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMinimumWithdrawal' title={localize('Minimum withdrawal')}>
-                <Money amount={payment_agent.min_withdrawal || ''} currency={payment_agent.currency} show_currency />
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMaximumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMaximumWithdrawal' title={localize('Maximum withdrawal')}>
-                <Money amount={payment_agent.max_withdrawal || ''} currency={payment_agent.currency} show_currency />
+            <PaymentAgentDetail icon='IcAccountTransfer' title={localize('Transfer limit')}>
```

Comment:
use `<localize />` component. We are in the process of deprecating `localize` function

Reply:
Alright, updated together with other implementations of `localize` in this file to avoid future problems